tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1,255 +1,887 @@
+
+
+error: \\"bublic\\" is not a field of \\"BucketProps\\"
+  --> ../../../examples/tests/invalid/struct_expansion.w:3:15
+  |
+3 | let bucket1 = new cloud.Bucket(bublic: false, public: true);
+  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ \\"bublic\\" is not a field of \\"BucketProps\\"
+
+
+error: Expected between 0 and 1 arguments but got 2 when instantiating \\"Bucket\\"
+  --> ../../../examples/tests/invalid/struct_expansion.w:7:15
+  |
+7 | let bucket2 = new cloud.Bucket(2, public: true);
+  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Expected between 0 and 1 arguments but got 2 when instantiating \\"Bucket\\"
+
+
+error: Expected type to be \\"BucketProps?\\", but got \\"num\\" instead
+  --> ../../../examples/tests/invalid/struct_expansion.w:7:32
+  |
+7 | let bucket2 = new cloud.Bucket(2, public: true);
+  |                                ^ Expected type to be \\"BucketProps?\\", but got \\"num\\" instead
+
+
+error: Expected 2 arguments but got 1
+   --> ../../../examples/tests/invalid/struct_expansion.w:11:3
+   |
+11 |   bucket1.put(file: \\"file.txt\\", \\"data\\");
+   |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Expected 2 arguments but got 1
+
+"
 `;
 
 exports[`try_no_catch_or_finally.w > stderr 1`] = `
-"Compilation failed with 1 error(s)
-Error at ../../../examples/tests/invalid/try_no_catch_or_finally.w:1:1 | Missing \`catch\` or \`finally\` blocks for this try statement"
+"error: Missing \`catch\` or \`finally\` blocks for this try statement
+  --> ../../../examples/tests/invalid/try_no_catch_or_finally.w:1:1
+  |  
+1 | / try {
+2 | |   print(\\"Hello World\\");
+3 | | }
+  | \\\\-^ Missing \`catch\` or \`finally\` blocks for this try statement
+
+"
 `;
 
 exports[`types_strings_arithmetic.w > stderr 1`] = `
-"Compilation failed with 5 error(s)
-Error at ../../../examples/tests/invalid/types_strings_arithmetic.w:1:10 | Expected type to be \\"num\\", but got \\"str\\" instead
-Error at ../../../examples/tests/invalid/types_strings_arithmetic.w:1:16 | Expected type to be \\"num\\", but got \\"str\\" instead
-Error at ../../../examples/tests/invalid/types_strings_arithmetic.w:4:14 | Expected type to be \\"num\\", but got \\"str\\" instead
-Error at ../../../examples/tests/invalid/types_strings_arithmetic.w:7:10 | Expected type to be \\"num\\", but got \\"str\\" instead
-Error at ../../../examples/tests/invalid/types_strings_arithmetic.w:13:10 | Expected type to be \\"num\\", but got \\"str\\" instead"
+"error: Expected type to be \\"num\\", but got \\"str\\" instead
+  --> ../../../examples/tests/invalid/types_strings_arithmetic.w:1:10
+  |
+1 | let e0 = \\"2\\" + \\"2\\";
+  |          ^^^ Expected type to be \\"num\\", but got \\"str\\" instead
+
+
+error: Expected type to be \\"num\\", but got \\"str\\" instead
+  --> ../../../examples/tests/invalid/types_strings_arithmetic.w:1:16
+  |
+1 | let e0 = \\"2\\" + \\"2\\";
+  |                ^^^ Expected type to be \\"num\\", but got \\"str\\" instead
```

Comment:
Ah, this exposes a pretty unfortunate problem with snapshots (character offsets not matching due to escape characters).

Not worth blocking this great feature, but we probably need to figure out something to see diagnostics as-is